astropy/units/tests/test_quantity_non_ufuncs.py

```diff
@@ -2813,6 +2836,11 @@ def test_all_arguments_reexposed(self, target, helper):
                         f"helper is not re-exposing argument {nt!r} properly: "
                         f"expected {kt}, got {kh}"
                     )
+                elif nt == "like":
```

Comment:
This could go with my suggestion.

Reply:
not completely though: since re-exposing `like` doesn't really make sense, I'm still keeping this branch (simplified, though)